src/mutations/member.ts

```diff
@@ -113,11 +113,8 @@ export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
 
   // mutation to update favorite items of given member
   queryClient.setMutationDefaults(MUTATION_KEYS.ADD_FAVORITE_ITEM, {
-    mutationFn: (payload) => {
-      const { memberId, itemId, extra } = payload;
-      extra.favoriteItems = extra.favoriteItems
-        ? extra.favoriteItems.concat([itemId])
-        : [itemId];
+    mutationFn: async (payload) => {
+      const { memberId, extra } = payload;
```

Comment:
What's the logic here? Has `extra` already been changed at this point? It doesn't seem so since you're changing it in `onMutate`.. I might be wrong but you never update the database actually.